### PR TITLE
Use generated map in listing pages

### LINKED ISSUE #559: EditEntry:Tabular:Use png for maps

### PR DESCRIPTION
**Pull request by AdityaKhatri**
Fix https://github.com/the-deep/deeper/issues/559
Depends on https://github.com/toggle-corp/react-store/pull/562
Depends on https://github.com/the-deep/server/pull/292

### CODE REVIEW

**Comment by frozenhelium** [comment]
Use SVG

File: src/components/geo/GeoViz/Region.js [hunk 1]
```diff
@@ -59,7 +59,7 @@ const boundsFilter = ['==', '$type', 'Polygon'];
 const pointsFilter = ['==', '$type', 'Point'];
 
 const colors = [
-    '#d5e7e2',
+    '#ffffff',
```
Comment (tnagorra) [inline src/components/geo/GeoViz/Region.js:62]: The server-side generated chart has a linear gradient for colors
The client-side chart uses n stops

**Comment by tnagorra** [comment]
- Cannot through image in Tabular Pane
- FilePreview component needs login in server

File: src/components/geo/GeoViz/Region.js (hunk 1 above)
Comment (frozenhelium) [reply src/components/geo/GeoViz/Region.js:62]: Should pull the color scheme from server

File: src/components/geo/GeoViz/Region.js [hunk 2]
```diff
@@ -104,90 +145,62 @@ export default class Region extends React.PureComponent {
...
                 ['get', 'count'],
-                ...iterableDetails,
+                0, colorScheme[0],
+                max, colorScheme[1],
             ],
             'fill-opacity': 0.9,
         });
     });
 
     calcBorderPaint = memoize(() => ({
```
Comment (frozenhelium) [inline src/components/geo/GeoViz/Region.js]: Memoize without parameter does not make sense

File: src/components/geo/GeoViz/styles.scss [hunk 3]
```diff
@@ -45,5 +45,26 @@
         position: absolute;
         bottom: var(--spacing-small);
         left: var(--spacing-small);
+        border: var(--width-separator-thin) solid var(--color-separator);
+        background-color: var(--color-foreground);
+        padding: var(--spacing-medium);
+        width: 240px;
```
Comment (frozenhelium) [inline src/components/geo/GeoViz/styles.scss]: ?

File: src/components/viewer/FilePreview/styles.scss [hunk 4]
```diff
@@ -0,0 +1,12 @@
+.not-found {
+    width: 100%;
+    height: 100%;
+}
+
+.image {
+    width: 100%;
+
+    &.svg-image {
```
Comment (frozenhelium) [inline src/components/viewer/FilePreview/styles.scss]: ?

**Pull request merged**